Inc/traceDecoder.h

```diff
@@ -143,6 +143,9 @@ struct TRACECPUState
 
     // Convinience, for debug reporting
     genericsReportCB report;
+
+    // Debugging
+    uint64_t overflows;       
```

Comment:
While using instruction tracing we had problems with overflows even with implicit tracing. Because there is no feedback I added an overflow counter and a debug message.